templates/default/fuse_seccomp.json

```diff
@@ -0,0 +1,380 @@
+{
+  "defaultAction":"SCMP_ACT_ERRNO",
+  "defaultErrnoRet":1,
+  "archMap":[
+    {
+      "architecture":"SCMP_ARCH_X86_64",
+      "subArchitectures":[
+        "SCMP_ARCH_X86",
+        "SCMP_ARCH_X32"
+      ]
+    }
+  ],
+  "syscalls":[
```

Comment:
What syscalls are needed? As far as I understand this allows everything!

Reply:
The default seccomp profile is this one https://github.com/moby/moby/blob/master/profiles/seccomp/default.json. I took some syscalls that require the CAP_SYS_ADMIN, https://github.com/moby/moby/blob/1c2d8c8663e63fe8ac19c56fc8e6f48915598a8a/profiles/seccomp/default.json#L591,  and added them to the ones that are allowed by default. My understanding is that the ones https://github.com/moby/moby/blob/1c2d8c8663e63fe8ac19c56fc8e6f48915598a8a/profiles/seccomp/default.json#L57 here are allowed by default.

Reply:
Your understanding is correct. The problem is that we can't blindly allow everything that CAP_SYS_ADMIN grants. Which syscalls are needed by fuse?

Reply:
These ones:  "clone", "mount", "umount", "umount2", "unshare"

Reply:
I suggest we take the default from https://github.com/moby/moby/blob/master/profiles/seccomp/default.json and then for clarity create another section with the required system calls, such as

```
{
			"names": [
				"clone",
				"mount",
				"umount",
                                "umount2",
                                "unshare"
			],
                        "comment": "Hopsworks hopsfs-mount required syscalls",
			"action": "SCMP_ACT_ALLOW"
		},